### PR TITLE
jenkins: Increase VM resources for feature_tests

### PR DESCRIPTION
**Pull request by fmuyassarov**
Increase number of cores, RAM and disk space for feature tests.

### CODE REVIEW

File: jenkins/scripts/integration_test.sh [hunk 1]
```diff
@@ -45,7 +45,12 @@ TEST_EXECUTER_VM_NAME="${TEST_EXECUTER_VM_NAME:-ci-test-vm-${VM_TIMELABEL}}"
 TEST_EXECUTER_PORT_NAME="${TEST_EXECUTER_PORT_NAME:-${TEST_EXECUTER_VM_NAME}-int-port}"
 if [ "${IMAGE_OS}" == "Ubuntu" ]
 then
+  if [ "${TESTS_FOR}" == "feature_tests" ]
+  then
+    TEST_EXECUTER_FLAVOR="${TEST_EXECUTER_FLAVOR:-8C-32GB-300GB}"
+  else
   TEST_EXECUTER_FLAVOR="${TEST_EXECUTER_FLAVOR:-4C-16GB-200GB}"
```
Comment (maelk) [inline jenkins/scripts/integration_test.sh:52]: indentation

Comment (fmuyassarov) [reply jenkins/scripts/integration_test.sh]: fixed

File: jenkins/scripts/integration_test.sh [hunk 2]
```diff
@@ -45,7 +45,12 @@ TEST_EXECUTER_VM_NAME="${TEST_EXECUTER_VM_NAME:-ci-test-vm-${VM_TIMELABEL}}"
 TEST_EXECUTER_PORT_NAME="${TEST_EXECUTER_PORT_NAME:-${TEST_EXECUTER_VM_NAME}-int-port}"
 if [ "${IMAGE_OS}" == "Ubuntu" ]
 then
-  TEST_EXECUTER_FLAVOR="${TEST_EXECUTER_FLAVOR:-4C-16GB-200GB}"
+  if [ "${TESTS_FOR}" == "feature_tests" ]
+  then
+    TEST_EXECUTER_FLAVOR="${TEST_EXECUTER_FLAVOR:-8C-32GB-300GB}"
```
[the remark this answers is not in the document]
Comment (fmuyassarov) [reply jenkins/scripts/integration_test.sh]: Done

File: jenkins/scripts/integration_test.sh [hunk 3]
```diff
@@ -43,6 +43,12 @@ DEFAULT_HOSTS_MEMORY="8192"
 VM_TIMELABEL="${VM_TIMELABEL:-$(date '+%Y%m%d%H%M%S')}"
 TEST_EXECUTER_VM_NAME="${TEST_EXECUTER_VM_NAME:-ci-test-vm-${VM_TIMELABEL}}"
 TEST_EXECUTER_PORT_NAME="${TEST_EXECUTER_PORT_NAME:-${TEST_EXECUTER_VM_NAME}-int-port}"
+
+if [ "${TESTS_FOR}" == "feature_tests" ]
+then
+  TEST_EXECUTER_FLAVOR="${TEST_EXECUTER_FLAVOR:-8C-32GB-300GB}"
```
Comment (furkatgofurov7) [inline jenkins/scripts/integration_test.sh:49]: Was not it 500GB disc space for proper testing?

Comment (fmuyassarov) [reply jenkins/scripts/integration_test.sh:49]: I think 300 GB is enough. But if turns out to be not enough during the tests, then we will increase it.

**Pull request merged**